uPortal-webapp/src/main/webapp/WEB-INF/flows/edit-portlet/portletConfig.jsp

```diff
@@ -473,13 +473,20 @@ PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
                                                         <i class="fa fa-info-circle"></i>
                                                     </a>
                                                 </td>
+                                                <td class="col-sm-3 text-nowrap">
+                                                    <spring:message code="edit.configure"/>
+                                                    <a href="javascript:void(0)" data-toggle="tooltip" data-placement="top" title="<spring:message code='edit.configure.tooltip'/>">
+                                                        <i class="fa fa-info-circle"></i>
```

Comment:
Is this a [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/)? Does it need ARIA treatment or an `alt=""` to reflect this?

Reply:
Because this is an image `alt` may not help.
a better fit might be
``` html
<i class="fa fa-info-circle" aria-hidden="true"></i>
<span class="sr-only">
  help
</span>
```

Reply:
I spoke with @ChristianMurphy about this, and am testing a fix for better ARIA treatment then what's currently setup.  Thanks for flagging it :)

Reply:
Slightly different then was was posted, but same idea.  Browse, Subscribe, and Configure info icons are tab-friendly and work with screenreaders now.  UP-4938 opened to change the rest of the portlet config checkboxes to match.